docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -122,6 +122,12 @@ It is recommended you do a [backup]({{< relref "doc/usage/backup-and-restore.en-
 If you have carried out the installation steps as described above, the binary should
 have the generic name `gitea`. Do not change this, i.e. to include the version number.
 
+To restart your gitea instance, we recommend to use SIGHUP signal. If you know your gitea PID use ```kill -1 $GITEA_PID``` otherwise you can use ```killall -1 gitea``` or ```pkill -1 gitea```
```

Comment:
tell about `systemctl restart gitea` first then explain the stuff if you dont use systemd

Reply:
Hello

New commit to present systemd restart.

Reply:
IMHO we should recommend the standard signal for termination, which is `SIGTERM` (`-15` or no number at all as it's the default signal for the `kill` command). Some systems will use `nohup` to start daemons and they will have `SIGHUP` masked (ignored). `kill -1 {pid}` will not work in those cases.